peekingduck/pipeline/nodes/input/utils/read.py

```diff
@@ -118,3 +118,13 @@ def resolution(self) -> Tuple[int, int]:
         width = self.stream.get(cv2.CAP_PROP_FRAME_WIDTH)
         height = self.stream.get(cv2.CAP_PROP_FRAME_HEIGHT)
         return int(width), int(height)
+
+    @property
+    def frame_count(self) -> float:
+        """ Get total number of frames of file
+
+        Returns:
+            int: number indicating frame count
+        """
+        num_frames = self.stream.get(cv2.CAP_PROP_FRAME_COUNT)
```

Comment:
It appears to work fine on images, but out of curiosity, what is the value of `cv2.CAP_PROP_FRAME_COUNT` on a single image? Does it return 1?

Reply:
yes, I believe so.